lib/pull-request/pr.ts

```diff
@@ -135,6 +135,13 @@ export interface AutoPullRequestProps extends AutoPullRequestOptions {
    * @default - no condition
    */
   condition?: string;
+
+  /**
+   * If any PR labeled with the given labels is still open, no new PR will be created
+   *
+   * @default - don't look at open PRs
+   */
+  readonly notIfLabeledPrOpen?: string[];
```

Comment:
Whaddya think of this?

```suggestion
  readonly skipIfOpenPrsWithLabels?: string[];
```

Reply:
Sounds good.